src/components/common/SearchBar/SearchBar.module.scss

```diff
@@ -13,6 +14,12 @@
   background-color: $primary-color-10-user;
   padding: var(--horizontal-spacing-normal);
   box-shadow: inset 0 -1px 0 0 $primary-color-10-user;
+  @include mobile-media {
+    order: -1;
+    margin-left: 0;
+    align-self: start;
+    margin-top: 14px;
+  }
 
```

Comment:
Let's make the search bar extend the full width for mobile :D
https://gyazo.com/2b351fc8992b5648c97986dc7c8a51a5